### ISSUE TITLE
why the value of 'masks_pred' have negative number?

### ISSUE THREAD

**Issue by yanzhou-li** [milesial/Pytorch-UNet#257]
when i used my own dataset to run  `train.py`, after run to `masks_pred = net(imgs)`, generating so many negative number,however, the value of `loss(batch)` is very low , is this normal?what happened?

**Comment by milesial** [comment]
This is normal. The output of the net are the raw unnormalized logits. The softmax normalization is included in the loss

**Comment by yanzhou-li** [comment]
> This is normal. The output of the net are the raw unnormalized logits. The softmax normalization is included in the loss

got it. thank you!
